### PR TITLE
[copy update] /security/fips

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated as per [doc](https://docs.google.com/document/d/1moTJCAtFXKD7ZXrxB-EB7GPVyZRTSIdkP-0BhPZl69M/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-12856.demos.haus/security/fips
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that the requested changes have been made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3659

### CODE REVIEW

File: templates/security/fips.html [hunk 1]
```diff
@@ -38,21 +38,21 @@ <h2 class="p-heading--4">FIPS 140 validated cryptography for Linux workloads on
...
       <hr class="u-sv1" />
       <p class="p-card__content">
-        U.S Federal agencies and anyone deploying systems and cloud services for Federal government agency use, whether directly or through contractors, are required to run workloads with FIPS 140 validated cryptography. FIPS 140 has also been adopted outside of the public sector in industries where data security is heavily regulated, such as financial services (PCI-DSS), healthcare (HIPAA), and other sectors. <a href="/cloud/public-cloud">Ubuntu Pro</a> and <a href="/pro">Ubuntu Advantage</a> provide FIPS 140 certified cryptographic packages.
+        U.S Federal agencies and anyone deploying systems and cloud services for Federal government agency use, whether directly or through contractors, are required to run workloads with FIPS 140 validated cryptography. FIPS 140 has also been adopted outside of the public sector in industries where data security is heavily regulated, such as financial services (PCI-DSS), healthcare (HIPAA), and other sectors. <a href="/cloud/public-cloud">Ubuntu Pro</a> provides FIPS 140 certified cryptographic packages.
```
Comment (pedoch) [inline templates/security/fips.html:41]: The `href` for the Ubuntu Pro anchor tag in the paragraph should go to `/pro`.

File: templates/security/fips.html
Comment (pedoch) [inline templates/security/fips.html:1]: For line 163, the asterisk (*) in the table header "UBUNTU 20.04 LTS
ON X86-64 AND IBM Z*" was removed in the copy docs.

File: templates/security/fips.html [hunk 2]
```diff
@@ -12,10 +12,10 @@
    <div class="col-8">
      <h1>FIPS for Ubuntu</h1>
      <h2 class="p-heading--4">FIPS 140 validated cryptography for Linux workloads on Ubuntu</h2>
-     <p>Developing and running Linux workloads for U.S. government regulated and high-security environments requires a long and expensive validation process. Reduce your accreditation timeline and pass on your validation costs with the FIPS 140 certified cryptographic packages of <a href="/cloud/public-cloud">Ubuntu Pro</a> and <a href="/pro">Ubuntu Advantage</a>.</p>
```
Comment (pedoch) [inline templates/security/fips.html:15]: The hyphen between "high" and "security" should not be deleted, that is what is on the copy doc.

**Review by pedoch — CHANGES_REQUESTED** [review]
Thanks @mtruj013! Nice work. Just a couple of minor changes.

File: templates/security/fips.html (hunk 2 above)
Comment (mtruj013) [reply templates/security/fips.html:15]: Resolved the comment on the doc since that's already what's live

**Comment by mtruj013** [comment]
Thanks @pedoch ! Could you approve please?

**Review by pedoch — APPROVED** [review]
LGTM!

**Pull request merged**